mask6.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2021 The Inet.Af AUTHORS. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package netaddr
+
+// mask6 are bitmasks with the topmost n bits of a
+// 128-bit number, where n is the array index.
+//
+// generated with https://play.golang.org/p/dGA3h0jrTry
+var mask6 = [...]uint128{
```

Comment:
maybe this should be a const string instead? the compiler might do a better job.

Reply:
We don't do constant-foldable lookups. I'm not sure it's worth the considerable reduction in readability. I'm open to being convinced otherwise by numbers, though.